components/picker/index.css

```diff
@@ -130,289 +195,239 @@ governing permissions and limitations under the License.
+  transition: color var(--mod-picker-animation-duration, var(--spectrum-picker-animation-duration)) ease-out;
+  flex-shrink: 0;
+  margin-inline-start: var(--mod-picker-spacing-icon-to-disclosure-icon, var(--spectrum-picker-spacing-icon-to-disclosure-icon));
 
-  &:hover {
-    background-color: var(--spectrum-picker-m-quiet-texticon-background-color-hover);
-    color: var(--spectrum-picker-m-texticon-text-color-hover);
+  color: var(--highcontrast-picker-icon-color-default, var(--mod-picker-icon-color-default, var(--spectrum-picker-icon-color-default)));
+
+  &:focus-visible {
+    color: var(--highcontrast-picker-icon-color-key-focus, var(--mod-picker-icon-color-key-focus, var(--spectrum-picker-icon-color-key-focus)));
+  }
+
+  &:active {
+    color: var(--highcontrast-picker-icon-color-default, var(--mod-picker-icon-color-active, var(--spectrum-picker-icon-color-active)));
   }
+}
 
-  &:focus-ring,
+.spectrum-Picker-validationIcon {
+  margin-inline-start: var(--mod-picker-spacing-text-to-alert-icon-inline-start, var(--spectrum-picker-spacing-text-to-alert-icon-inline-start));
+}
+
+.spectrum-Picker-label ~ .spectrum-Picker-menuIcon {
+  margin-inline-start: var(--mod-picker-spacing-text-to-icon, var(--spectrum-picker-spacing-text-to-icon));
+}
+
+.spectrum-Picker + .spectrum-Popover--bottom.is-open {
+  transform: translateY(var(--mod-picker-spacing-picker-to-popover, var(--spectrum-picker-spacing-picker-to-popover)));
+  transform: translateY(var(--mod-component-to-menu-medium, var(--spectrum-component-to-menu-medium)));
+}
+
+.spectrum-Picker-popover--quiet {
+  margin-inline-start: calc(
+    -1 * calc(
+      var(--mod-picker-popover-quiet-offset-x, var(--spectrum-picker-popover-quiet-offset-x)) + var(--mod-picker-border-size, var(--spectrum-picker-border-size))
+    )
+  );
+}
+
+.spectrum-Picker--quiet {
+  border: none;
+  border-radius: 0;
+  padding-inline: var(--mod-picker-spacing-edge-to-text-quiet, var(--spectrum-picker-spacing-edge-to-text-quiet));
+  margin-block-start: var(--mod-picker-spacing-label-to-picker-quiet, var(--spectrum-picker-spacing-label-to-picker-quiet));
+
+  color: var(--highcontrast-picker-font-color-default, var(--mod-picker-font-color-default, var(--spectrum-picker-font-color-default)));
+  background-color: transparent;
+
+  &:focus-visible,
   &.is-focused {
-    background-color: var(--spectrum-picker-quiet-background-color-key-focus);
-    box-shadow: 0 2px 0 0 var(--spectrum-picker-quiet-border-color-key-focus);
+    box-shadow: 0 var(--mod-picker-focus-indicator-size, var(--spectrum-picker-focus-indicator-size)) 0 0 var(--highcontrast-picker-focus-indicator-color, var(--spectrum-picker-focus-indicator-color));
 
-    &.is-placeholder {
-      color: var(--spectrum-picker-m-quiet-texticon-placeholder-text-color-key-focus);
-    }
-    .spectrum-Picker-menuIcon {
-      color: var(--spectrum-picker-m-texticon-icon-color-key-focus)
-    }
     &::after {
       box-shadow: none;
```

Comment:
Why is the focus state for quiet being applied as a box shadow on the component instead of on the ::after as it is on the standard component?

Also, focus indicator is missing the focus indicator gap. It is hugging the component.

Reply:
This was a mistake, updated.